src/PopoverTrigger.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module PopoverTrigger
+ */
+class PopoverTrigger extends React.Component {
+
+	render() {
+		const {
+			children,
+			className,
+			handleClick,
+			handleKeyDown,
+			handleBlur,
```

Comment:
The React convention is to name event handlers with an `on..` prefix to match up with the `prop` they get assigned to, e.g. `onClick={onClick}` - just a tiny bit of extra consistency.